tools/grpcurl-query-ingesters/download-chunks-from-ingesters.sh

```diff
@@ -22,14 +25,14 @@ for POD in $PODS; do
   # Wait some time
   sleep 5
 
-  cat query.json | grpcurl \
+  cat "$SCRIPT_DIR/download-chunks-from-ingesters-query.json" | grpcurl \
     -d @ \
     -H "X-Scope-OrgID: $MIMIR_TENANT_ID" \
   	-proto pkg/ingester/client/ingester.proto \
-  	-import-path . \
-  	-import-path ./vendor \
+  	-import-path "$SCRIPT_DIR" \
+  	-import-path "$SCRIPT_DIR/vendor" \
```

Comment:
Tabs -> spaces (was missed during my first review, not sure why)
```suggestion
    -import-path "$SCRIPT_DIR" \
    -import-path "$SCRIPT_DIR/vendor" \
```

Reply:
Good catch, fixed in f60bc81